Adding-Snippets-In-Sublime-Text.md

```diff
@@ -0,0 +1,41 @@
+# Adding Snippets To Your Sublime Text Installation
+
+If you are a user of sublime text (I hope you are, it kicks ass. You can get it [here](https://www.sublimetext.com/)), then you may have
+already noticed some of the tab completion capabilities. For example, when you are opening anhtml tag such as `<p>`, it automatically closes
```

Comment:
There is a typo where it says "when you are opening anhtml tag such as" it should read "when you are opening a html tag such as"